Src/HALAL/Services/Communication/UART/UART.cpp

```diff
@@ -35,7 +35,7 @@ void UART::start(){
 		}
 }
 
-bool UART::transmit_next_packet(uint8_t id, RawPacket& packet){
+bool UART::transmit(uint8_t id, uint8_t data){
```

Comment:
```suggestion
bool UART::transmit(uint8_t id, uint8_t data){
    UART::transmit(id, &data, 1);
}
```
Si esto compila, en vez de copiar y pegar la funcion dos veces puedes llamar a la funcion general en el caso especifico